Community/ideAlarm/automation/lib/python/community/idealarm/__init__.py

```diff
@@ -304,16 +305,23 @@ def onToggleSwitch(self, itemName):
         Called whenever an alarm arming mode toggle switch has been switched.
         '''
         newArmingMode = None
+        pinCode = getItemValue(itemName, 0)
+        if pinCode == 0:
```

Comment:
Wouldn't this make a PIN code a requirement for arming/disarming? I think this can be removed.

Reply:
I need to slightly change it indeed, also need to check if self.pinCode is None (pinCode not enabled)